docs/fullDocDraft.md

```diff
@@ -0,0 +1,361 @@
+
+G4CMP addresses these issues via the G4CMPSecondaryProduction and
+G4CMPEnergyPartition classes, allowing for charge tracking at the lower energy
+levels used in low-temperature measurement that is otherwise impossible in
+Geant4.
+
+[//]: # (ibid, slide 48)
+
+The Relative magnitude of $ dE/dx $ versus NIEL for ions depends on the charge
+and mass of the projectile as well as atomic number and mass of the crystal
+atoms. The ionization yield is computed by taking $ dE/dx $ as a fraction of the
+total. G4CMP does this in its code for ion hits, but it is now done
+automatically in Geant4 10.7: the issue is addressed via forward-compatibility,
+because G4CMP will not recalculate the yield in the case of non-zero NIEL. This
+process remains in the G4CMPLindhardNIEL and G4CMPLewinSmithNIEL classes.
+
+[//]: # (ibid, slide 49)
+
+## QET Physics
+
+G4CMP is primarily used for simulations in the material of a detector crystal,
+but it also supports some methods and classes for simulating detector response
+to events within the crystal. Here, we cover some of its applications in that
+realm.
+
+### Kaplan Quasiparticle Model
+
+This model is used for thin-film superconductors common in cryogenic electronic
+sensors, such as TESs or QETs, KIDs, and qubits. These sensors are populated
+with Cooper pairs, which can be broken upon phonon absorption into electron
+quasiparticles. For small films, quasiparticles transport faster than the
+thermal response. As for Crystal properties, aluminum film parameters are
+supplied with the package via a Geant4 properties table, including film
+thickness, Cooper pair gap energy (2Δ), $ v_{sound} $, and phonon lifetime.
+
+G4CMP models energy transfer, QP transport, and phonon re-emission via the
+G4CMPKaplanQP class.
+
+[//]: # (ibid, slide 50)
+
+### G4CMPKaplanQP
+
+In G4CMPKaplanQP, the simulation of the Kaplan Quasiparticle Model is treated as
+instantaneous; it iterates to find the equilibrium state and contains no
+time-dependent information.
+
+Substrate phonon absorption in the thin film is modeled by using the mean free
+path and thickness to find the probability of this occurring. In particular,
+
+$$
+P=e^{\frac{-4d}{MFP}}, \ MFP = \frac{v_{sound}}{\tau(E)}, \
+\tau(E)=\frac{\tau_0}{1+\delta \tau(E/\Delta-2)}
+$$
+
+where P is the probability of substrate phonon absorption, d is the film
+thickness, MFP is the mean free path, and 𝛕 is the phonon lifetime.
+
+Phonon energy in these simulations goes to break Cooper pairs, and becomes the
+QP energy. QP or phonon energy is absorbed into the tungsten TES, at which time
+some QP energy goes back into phonons via QP “decay” (emission), and some phonon
+energy is re-emitted back into the substrate. In G4CMPKaplanQP, the processing
+loop ends when the available phonon energy is zero. This model is not suitable
+for “bare” films that lack an attached energy absorber.
+
+[//]: # (ibid, slide 51)
+
+### Phonon Readout Model
```

Comment:
This is not part of G4CMP.  It's in the SuperSim/G4DMC package, and shouldn't be mentioned here.